### ISSUE TITLE
Better support R users

### ISSUE THREAD

**Issue by sidneymbell** [chanzuckerberg/cellxgene#634]
**tl;dr -- I'd love thoughts and discussion around this. I suspect this has several components to it; one big component of this is input data formats. I've taken some notes below about what (I think?) that landscape looks like based on some initial investigations.** 

In targeting the loop between experimental and computational users, it would allow us to support many more scientists if `launch` were able to support the output of seurat and bioconductor. 

For improving data ingress, I'd vote for implementing this in `launch` rather than `prepare` simply because if someone is already working in seurat/bioconductor (i.e., in an analytical loop), I assume they don't need to use `prepare` to do normalization, embedding, etc. I think this is also where we'd most likely be intercepting pairs of experimental/computational users. 

This does get a little tricky. In exploring and consulting with Angela Pisco and Olga Botvinnik, it appears that both seurat and bioconductor mostly just output `rds` and `rdata` objects. While there are [converters](https://satijalab.org/seurat/conversion_vignette.html) between the `seurat` <--> `SingleCellExperiment` R objects --> `h5ad`, these are still under development and nontrivial to work with. 

In the future, I suspect supporting `launch` from `rdata`/`rds` will be necessary. But, it does seem like a larger and more complex chunk of work than I had anticipated due to the absence of an intermediate data file structure.

In the meantime, I'd suggest implementing support for other file formats already supported by `anndata.read` as detailed [here](https://github.com/chanzuckerberg/cellxgene/issues/635).

**Comment by csweaver** [comment]
So I think there are two separate issues here:
1. Ease the burden for R users who want to use cellxgene as much as possible
2. Support loading of alternative file-formats that scanpy already supports into cellxgene (ex. mtx, loom, hdf5)

If (1) is hard, I think it's reasonable to start on (2). But I think it should be a separate ticket. I'm really hoping that it if we update cellxgene to use the correct load method and that it will just work. And then we can reach the R community and other users.

**Comment by sidneymbell** [comment]
Ah, interesting. @csweaver -- can you say more about which bit should be a separate ticket? Do you mean a separate ticket for future support of rdata / rds? (Happy to do so, just want to make sure I understand).

**Comment by csweaver** [comment]
I think that supporting the R ecosystem and supporting input from other file formats are going to be two separate tickets. For this ticket the title matchings the supporting input, but the body matches supporting R. And it seems like supporting R is a more involved process which deserves its own prioritization for roadmapping and releases.

**Comment by fidelram** [comment]
seurat has a function to save as AnnData. I think this works in Seurat but
not yet in Seurat3. Using this function, a file that can be directly used
with cellxgene can be loaded.

**Comment by sidneymbell** [comment]
@csweaver - That makes sense. I'll update and split these out, thanks.

@fidelram - Thanks so much, Fidel! 
To clarify, are you referring to the `Convert` function implemented [here](https://github.com/satijalab/seurat/blob/master/R/conversion.R)? 
Also, are there any other intermediate file formats (i.e., that generally contain both normalized values and attached metadata) that you'd like to see supported as an R user?

**Comment by signechambers1** [comment]
Closing as a duplicate of [#1841 ](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/cellxgene/1841)
